pkg/controller/util/node/controller_utils.go

```diff
@@ -97,6 +97,7 @@ func DeletePods(kubeClient clientset.Interface, recorder record.EventRecorder, n
 		if err := kubeClient.CoreV1().Pods(pod.Namespace).Delete(pod.Name, nil); err != nil {
 			return false, err
 		}
+		recorder.Eventf(&pod, v1.EventTypeNormal, "NodeControllerEviction", "Successfully deleted Pod %s from Node %s", pod.Name, nodeName)
```

Comment:
We don't know if it's successful deletion at that point, we need to wait for ack from kubelet (and NC is not waiting for that). Additionally it violates Event style guide: https://github.com/kubernetes/community/blob/master/contributors/devel/event-style-guide.md